app/components/Views/QRScanner/styles.ts

```diff
@@ -1,40 +1,60 @@
-import { StyleSheet } from 'react-native';
+/* eslint-disable react-native/no-color-literals */
+import { StyleSheet, TextStyle } from 'react-native';
 import { Theme } from '../../../util/theme/models';
+import { colors } from '../../../styles/common';
 
 const createStyles = (theme: Theme) =>
   StyleSheet.create({
     container: {
-      flex: 1,
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      height: '100%',
+      width: '100%',
       backgroundColor: theme.brandColors.black,
     },
     preview: {
       flex: 1,
+      width: '100%',
+      height: '100%',
+      position: 'absolute',
+      zIndex: 0,
     },
-    innerView: {
-      flex: 1,
+    overlayContainerColumn: {
+      display: 'flex',
+      width: '100%',
+      height: '100%',
+      flexDirection: 'column',
+      position: 'absolute',
+      zIndex: 1,
     },
-    closeIcon: {
-      marginTop: 20,
-      marginRight: 20,
-      width: 40,
-      alignSelf: 'flex-end',
-      color: theme.brandColors.white,
+    overlayContainerRow: {
+      display: 'flex',
+      flexDirection: 'row',
+    },
+    overlay: {
+      flex: 1,
+      flexBasis: 0,
+      backgroundColor: colors.overlay,
```

Comment:
We do not need to import colors at the styles file, since they are passed on the theme object
```suggestion
      backgroundColor: theme.colors.background,
```

Reply:
We actually do, since these colors are not theme-constrained

Reply:
Should we modify the theme?

Reply:
Isn't the theme constrained by dark/light modes?